### DISCUSSION TITLE
Secman V3

### DISCUSSION THREAD

**Discussion by abdfnx** [scmn-dev/secman#132] [Ideas]
# Secman V3 with Go

> now secman cli with typescript is good and nice, but it's slowly on startup and needs nodejs if you install secman cli

so, our next move is create the V3 of secman cli, we'll replace `TypeScript` and `Node.js` with `Golang` because it's faster, lightweight, and doesn't depend on nodejs

## What we'll do about secman v2 commands ?

> we're going to use the same Secman CLI V2 commands, we'll just convert from **typescript** to **golang**

**Comment by Timothee-Cardoso** [comment]
![cj](https://user-images.githubusercontent.com/80710181/149752952-58ce08e9-045f-4de0-89c2-f740e6c3ca23.jpg)

**Reply by abdfnx** [reply]
I know @Timothee-Cardoso , but we have to give our users the best experience 👍
